lib/ansible/modules/system/filesystem.py

```diff
@@ -384,24 +404,31 @@ def main():
 
```

Comment:
Should not `filesystem.get_uuid(dev)` be called here (then remove all other calls only except the one after `filesystem.create`) ?